Lib/test/test_dis.py

```diff
@@ -535,6 +579,17 @@ def test_disassemble_generator(self):
         gen_disas = self.get_disassembly(_g(1))  # Disassemble generator itself
         self.assertEqual(gen_disas, gen_func_disas)
 
+    def test_disassemble_async_generator(self):
+        agen_func_disas = self.get_disassembly(_ag) # Disassemble async generator function
+        agen_disas = self.get_disassembly(_ag(1)) # Disassemble async generator itself
+        self.assertEqual(agen_disas, agen_func_disas)
+
+    def test_disassemble_coroutine(self):
+        coro_func_disas = self.get_disassembly(_co) # Disassemble coroutine function
```

Comment:
Another nit: two spaces between code and `#` -- people who use linters will see this line highlighted.